symplyphysics/laws/electricity/circuits/capacitance_of_spherical_capacitor.py

```diff
@@ -6,69 +6,83 @@
 filled with a dielectric medium. See `Figure`_.
 
 .. _Figure: http://hyperphysics.phy-astr.gsu.edu/hbase/electric/capsph.html
+
+**Links:**
+
+#. `Spherical capacitor <http://hyperphysics.phy-astr.gsu.edu/hbase/electric/capsph.html>`__.
 """
 
 from sympy import (Eq, solve, pi)
 from symplyphysics import (
     units,
     Quantity,
-    Symbol,
+    SymbolNew,
     validate_input,
     validate_output,
+    quantities,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.electricity import (
+    electric_field_outside_charged_sphere as _electric_field_law,
+    voltage_is_line_integral_of_electric_field as _voltage_law,
+    capacitance_from_charge_and_voltage as _capacitance_law,
 )
 
-capacitance = Symbol("capacitance", units.capacitance)
+capacitance = SymbolNew("C", units.capacitance)
 """
 Capacitance of the capacitor.
-
-Symbol:
-    :code:`C`
 """
 
-absolute_permittivity = Symbol("absolute_permittivity", units.capacitance / units.length)
-r"""
+absolute_permittivity = SymbolNew("epsilon", units.capacitance / units.length, display_latex="\\varepsilon")
+"""
 Absolute permittivity of the medium between the spheres.
-
-Symbol:
-    :code:`epsilon`
-
-Latex:
-    :math:`\varepsilon`
 """
 
-inner_radius = Symbol("inner_radius", units.length)
-r"""
+inner_radius = SymbolNew("r_in", units.length, display_latex="r_\\text{in}")
+"""
 Radius of the inner sphere.
-
-Symbol:
-    :code:`r_in`
-
-Latex:
-    :math:`r_\text{in}`
 """
 
-outer_radius = Symbol("outer_radius", units.length)
-r"""
+outer_radius = SymbolNew("r_out", units.length, display_latex="r_\\text{out}")
+"""
 Radius of the outer sphere.
-
-Symbol:
-    :code:`r_out`
-
-Latex:
-    :math:`r_\text{out}`
 """
 
 law = Eq(
     capacitance, 4 * pi * absolute_permittivity * inner_radius * outer_radius /
     (outer_radius - inner_radius))
-r"""
-:code:`C = 4 * pi * epsilon * r_in * r_out / (r_out - r_in)`
+"""
+:laws:symbol::
 
-Latex:
-    .. math::
-        C = \frac{4 \pi \varepsilon r_\text{in} r_\text{out}}{r_\text{out} - r_\text{in}}
+:laws:latex::
 """
 
+# Derive law for spherical capacitor filled with vacuum
+
+_distance = _voltage_law.distance
+
+_electric_field_expr = _electric_field_law.law.rhs.subs({
+    _electric_field_law.charge: _capacitance_law.charge,
+    _electric_field_law.distance: _distance,
+})
+
+_voltage_expr = _voltage_law.law.rhs.subs({
+    _voltage_law.electric_field_component(_distance): _electric_field_expr,
+    _voltage_law.initial_distance: inner_radius,
+    _voltage_law.final_distance: outer_radius,
+}).doit()
+
+# Multiply by -1 to make the ratio positive
+_capacitance_expr = -1 * _capacitance_law.definition.rhs.subs({
```

Comment:
Maybe use `outer_radius` for initial distance to avoid negation?

Reply:
Okay, I'll add a todo about cloning symbols with correct assumptions.